auth/src/main/java/com/firebase/ui/auth/ui/idp/CredentialSignInHandler.java

```diff
@@ -82,9 +84,10 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                             .addOnFailureListener(new OnFailureListener() {
                                 @Override
                                 public void onFailure(@NonNull Exception e) {
-                                    // TODO: What to do when signing in with Credential fails
-                                    // and we can't continue to Welcome back flow without
-                                    // knowing providers?
+                                    mHelper.finishActivity(
+                                            mActivity,
+                                            ResultCodes.CANCELED,
+                                            IdpResponse.getErrorCodeIntent(ErrorCodes.UNKNOWN_ERROR));
```

Comment:
I think any solution is better than none at this point since we don't even notify the user that anything has happened (failing silently is almost always a bad idea IMO). Another possibility would be to show a toast to the user, but this solution could in theory give more control to the dev if we refactor the `ErrorCodes`. @samtstern What are your thoughts?